mws/mws.py

```diff
@@ -229,6 +236,13 @@ def make_request(self, extra_data, method="GET", **kwargs):
         parsed_response.response = response
         return parsed_response
 
+    def set_proxies(self):
```

Comment:
I think it would make sense to call this `get_proxies` or something similar, as we are not setting anything here on the instance, rather just making the dictionary and returning it